packages/utilities/src/functional/number.ts

```diff
@@ -0,0 +1,3 @@
+export function number<T>(value: T, fallback: number): number {
```

Comment:
generic param is pointless and the name of function is not self explaining

Reply:
How about renaming to e.g. `numberWithFallback`?

Reply:
Renamed to `numberOrFallback()`.